src/getHumanAge.test.js

```diff
@@ -7,4 +7,53 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return zeros if catAge and dogAge = 0', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result)
+      .toEqual([0, 0]);
+  });
+
+  test('should return zeros if catAge and dogAge < 15', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result)
+      .toEqual([0, 0]);
+  });
+
+  test('should return 1\'s if catAge and dogAge are between 15 and 23', () => {
+    const result = getHumanAge(23, 23);
+
+    expect(result)
+      .toEqual([1, 1]);
+  });
+
+  test('should return first number 2 if catAge are between 24 and 27', () => {
+    const result = getHumanAge(27, 0);
+
+    expect(result)
+      .toEqual([2, 0]);
+  });
+
+  test('should return second number 2 if dogAge are between 24 and 28', () => {
```

Comment:
not clear, what do we check? we should check 24, 27, 28, and 29 for BOTH parameters